test/benchmarks_test.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:convert' show JsonEncoder;
+import 'dart:io';
+
+import 'package:gallery/benchmarks/common.dart';
+import 'package:test/test.dart';
+import 'package:web_benchmarks/server.dart';
+
+final listOfBenchmarks = <String>[
+  galleryStudiesPerf,
+  galleryUnanimatedPerf,
+  galleryAnimatedPerf,
+  galleryScrollPerf,
+];
+
+Future<void> main() async {
+  test('Can run a web benchmark', () async {
+    print ('Starting web benchmark tests ...');
+
+    final taskResult = await serveWebBenchmark(
+      benchmarkAppDirectory: Directory.current.absolute,
+      entryPoint: 'lib/benchmarks/client.dart',
+      useCanvasKit: false,
+    );
+
+    print ('Web benchmark tests finished.');
+
+    for (final benchmarkName in listOfBenchmarks) {
+      for (final metricName in <String>[
+        'preroll_frame',
+        'apply_frame',
+        'drawFrameDuration',
+      ]) {
+        for (final valueName in <String>[
+          'average',
+          'outlierAverage',
+          'outlierRatio',
+          'noise',
+        ]) {
+          expect(
```

Comment:
This wont test against other metric + value names that might exist. If that needs to be tested, create another expect for `taskResult.scores.size/length, equals(3 * 4)` (or if the metricName and valueName lists are extracted so you are able to programmatically grab the lengths to multiply)

Reply:
Done. Thank you!